doc/source/config.rst

```diff
@@ -260,6 +260,20 @@ correction.
 
 .. _component_configuration:
```

Comment:
This label needs to be moved back to the correct section (the `Component Configuration` section below the temporary directory section).

Reply:
Done, thanks for checking!